README.md

```diff
@@ -1,17 +1,56 @@
 # Capybara Workout
-I created [this site](//capybaraworkout.herokuapp.com/) as a playground for students at Makers Academy to learn to use Capybara to send instructions to an automated browser.
 
-### Tech
-I took the opportunity to explore functional javascript with the [Ramda](https://github.com/ramda/ramda) library.
-```js
-R.compose(_.setHTMLofElement(successMsg), _.getClickedElement)
+![Capybara Workout Image](./app/public/images/CapybaraWorkout.jpg)
+
+## Running locally
+
+Choose between:
+```bash
+    rackup # runs on http://localhost:9292/
+    # or
+    ruby app/app.rb # runs on http://localhost:4567
+```
+
+This will bring you to the instructions for interacting with the app. Then once you have IRB working with Capybara and you're ready to start the activity:
+
+```bash
+    visit('http://localhost:<port>/workout')
+```
+
+## Development
+
+To run tests, I recommend installing [chromedriver](https://formulae.brew.sh/cask/chromedriver):
+
+```bash
+brew install --cask chromedriver
+```
+
+To contribute to CSS, please use Sass. You can install Dart Sass by running:
+
+```bash
+brew install sass/sass/sass
+```
+
+and then run the following watch command to automagically compile your changes down to CSS:
```

Comment:
I don't think anyone at this point would necessarily understand what sass is or what they're installing / watching here. Maybe even just a small link like, "You can learn more about SASS *here*" would be helpful here. Although I also suppose they could just google that themselves. It just feels like this and the chromedriver step are kind of just like "do this and don't ask why"

Reply:
Hey @ryanpierre! Thanks for the comment — this isn't really meant to be student-facing. When deployed, the devs will be directed to the website and will interact with the page elements using Capybara from their terminal. The `README` instructions are purely for staff to use/maintain.